xilem/src/pod.rs

```diff
@@ -1,38 +1,27 @@
 // Copyright 2025 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-use masonry::core::{
-    FromDynWidget, Properties, Widget, WidgetId, WidgetMut, WidgetOptions, WidgetPod,
-};
+use masonry::core::{FromDynWidget, NewWidget, Widget, WidgetMut};
 
 use crate::ViewCtx;
 use crate::core::{Mut, SuperElement, ViewElement};
 
 /// A container for a yet to be inserted [Masonry](masonry) widget
 /// to be used with Xilem.
 ///
-/// This exists for two reasons:
-/// 1) The nearest equivalent type in Masonry, [`WidgetPod`], can't have
-///    [Xilem Core](xilem_core) traits implemented on it due to Rust's orphan rules.
-/// 2) `WidgetPod` is also used during a widget's lifetime to contain its children,
-///    and so might not actually own the underlying widget value.
-///    When creating widgets in Xilem, layered views all want access to the - using
-///    `WidgetPod` for this purpose would require fallible unwrapping.
+/// This exists because the nearest equivalent type in Masonry, [`NewWidget`], can't have
+/// [Xilem Core](xilem_core) traits implemented on it due to Rust's orphan rules.
+///
+/// If changing transforms of widgets, make sure to use [`transformed`]
+/// (or [`WidgetView::transform`]).
+/// This has a protocol to ensure that multiple views changing the
+/// transform interoperate successfully.
+///
+/// [`transformed`]: crate::view::Transformed
+/// [`WidgetView::transform`]: crate::view::transformed
 #[expect(missing_docs, reason = "TODO - Document these items")]
 pub struct Pod<W: Widget + FromDynWidget + ?Sized> {
-    pub widget: Box<W>,
```

Comment:
Losing this field in Xilem is unfortunate, because it means that you can no longer have composing views for setting fields (other than transform/disabled/properties).
I see that since https://github.com/linebender/xilem/pull/837, this has been boxed, which interacts pretty poorly with the builder methods (you have to unbox and rebox at every layer).